packages/components/src/ConditionalFilter/GroupFilter.tsx

```diff
@@ -155,9 +156,14 @@ const GroupFilter: React.FunctionComponent<GroupFilterProps> = (props) => {
   };
 
   const handleClickOutside = (event: MouseEvent) => {
-    if (isOpen && !menuRef.current?.contains(event.target as Node)) {
-      setIsOpen(false);
-    }
+    if (!isOpen) return;
+
```

Comment:
Nit pick, could you add a comment here just explaining that 
`//If you click insight the input or menu, function does nothing, otherwise, close the menu`

This is more so just if someone doesnt understand what this is checking. I havent seen code similar to this in any of the repos

Reply:
@adonispuente I was thinking about it and will follow approach that when code is clear there is no need to comment it. Having there clear variable names like `clickedInsideInput` and `setIsOpen(false)` also directly tells what's going on.